pkg/sql/conn_executor.go

```diff
@@ -3206,6 +3213,52 @@ func (ex *connExecutor) runPreCommitStages(ctx context.Context) error {
 	return nil
 }
 
+func (ex *connExecutor) handleCommitWithCausalityToken(
+	ctx context.Context, res RestrictedCommandResult,
+) (fsm.Event, fsm.EventPayload, error) {
+	res.ResetStmtType((*tree.CommitTransaction)(nil))
+	err := ex.commitSQLTransactionInternal(ctx)
+	if err == nil {
+		res.SetColumns(ctx, colinfo.ResultColumns{
+			{
+				Name:   "causality_token",
```

Comment:
One area where a new user to the system can get hung up is all of the different ways that these timestamps are referred to - `crdb_internal_mvcc_timestamp`, `cluster_logical_timestamp`, "hybrid-logical clock timestamps", _etc_. While this result column name is specific and correct, it does add another descriptor to this concept and require that the user know how to connect the dots.

Reply:
Fair, let's see what other folks say regarding names. I feel no strong attachment towards this one.